pyemma/coordinates/api.py

```diff
@@ -1414,9 +1415,113 @@ def vamp(data=None, lag=10, dim=None, scaling=None, right=True, ncov_max=float('
     return res
 
 
+def tica_nystroem(data, lag, max_columns,
+                  dim=-1, var_cutoff=0.95, epsilon=1e-6,
+                  stride=1, skip=0, reversible=True, ncov_max=float('inf'),
+                  initial_columns=None, nsel=1, neig=None):
+    r""" Sparse sampling implementation [1]_ of time-lagged independent component analysis (TICA) [2]_, [3]_, [4]_.
+
+    Parameters
+    ----------
+    data : ndarray (T, d) or list of ndarray (T_i, d) or a reader created by
```

Comment:
the docstring says data is optional, but it isn't. All of the other estimators in coordinates api support just the construction of the object without estimation.